### ISSUE TITLE
Rotation returns wrong values on 64 bit systems

### ISSUE THREAD

**Issue by The-Michael-R** [The-Michael-R/Nuimo-Linux-C-SDK#3]
Running the code on a ArchLinux with 64Bit shows wrong values (rotation left starts at ~ -300, and at higher speeds both directions show unexpected values.

**Comment by larsblumberg** [comment]
Hi Michael, is this an issue with your SDK or Nuimo reporting wrong rotation values for you?

**Comment by The-Michael-R** [comment]
Hi Lars,
no, everything the Nuimo sends is fine. Here turning counter clock wise:
```
signal time=1475603194.840445 sender=:1.67 -> destination=(null destination) serial=580 path=/org/bluez/hci0/dev_xx_xx_xx_xx_xx_xx/service001b/char0025; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.bluez.GattCharacteristic1"
   array [
      dict entry(
         string "Value"
         variant             array of bytes [
               f5 ff
            ]
      )
   ]
   array [
   ]
```
And with the changed code the code returns the right values:
```
---  const char *value;
+++  const unsigned char *value;
[...]
  case NUIMO_ROTATION :
---    number = value[1] * 256 + value[0];
+++    number = ((value[1] & 255) << 8) + (value[0] & 255);
       direction = number > 0 ? NUIMO_ROTATION_LEFT : NUIMO_ROTATION_RIGHT;
    break;
```
As this is not a 32/64Bit issue, I actually wonder why the RasPi didn't show this error.

**Comment by The-Michael-R** [comment]
Fixed with commit [e67b679](https://github.com/The-Michael-R/Nuimo-Linux-C-SDK/commit/e67b67952e130bc1582b9a6c95d76c11ca43b5c8).

**Comment by larsblumberg** [comment]
Awesome!
